### DISCUSSION TITLE
how to paste text from keyboard

### DISCUSSION THREAD

**Discussion by jakepoolefin** [AutomaApp/automa#1798] [Q&A]
how to paste text from keyboard?

Form doesn't work for text input

Enter keys to send control+v to paste doesn't work either.

**Comment by 3Deyedoc** [comment]
Where are you copying the text from? If you can use the get text block then you can save the text to a variable and then use the forms block to enter the text into the appropriate spot. use {{ variables.text }} where "text" is whatever you named the variable in the get text. https://www.youtube.com/watch?v=mbs_mjzbxsk&t

**Reply by jakepoolefin** [reply]
It does not work.  Some websites nothing works!  Not forms, not triggers, not press keys, nothing.
Except the physical keyboard and mouse.

If I press control + v on the keyboard it works, but nothing in Automa works

How do I simulate the physical actions of pressing control+v in Automa to send text?  Then hitting reply?
